pkg/factorymodule/protobuf.go

```diff
@@ -0,0 +1,42 @@
+package factorymodule
+
+import (
+	"github.com/filecoin-project/mir/pkg/events"
+	"github.com/filecoin-project/mir/pkg/pb/eventpb"
+	"github.com/filecoin-project/mir/pkg/pb/factorymodulepb"
+	t "github.com/filecoin-project/mir/pkg/types"
+)
+
+func FactoryNewModule(
```

Comment:
Word `Factory` is redundant, it's already in the package name (`factorymodule.FactoryNewModule` looks bad).
Also, `module` in the package name feels a bit excessive too.

Reply:
Also, to avoid name collisions and confusion w.r.t. what the function does, I'd move event constructors to a subpackage like `factoryevents` (I usually call such a subpackage just `events` and expect it to be imported like `factoryevents "github.com/filecoin-project/mir/pkg/factory/events"`, but it's a matter of taste).

Reply:
Agree, it's cleaner this way. Fixed.